x509/common_test.go

```diff
@@ -201,7 +201,7 @@ func runTest(t *testing.T, verifyFiles func(factoryCa, tlsCert, onlineCa, offlin
 		crl, err := x509.ParseRevocationList(pemToDer(t, crlPem))
 		require.Nil(t, err)
 		require.NotNil(t, crl)
-		assert.Equal(t, factoryCa.Subject, crl.Issuer)
+		assert.Equal(t, factoryCa.Subject.String(), crl.Issuer.String())
```

Comment:
This check always passes for me locally either way.
But due to some CI magic, Golang always reorders pkix.Name attributes of the CRL subject to be reversed.

So, you might see a couple of random commits trying to fix this line.

Reply:
Passed on 3 re-runs, so I'm going to squash and consider it fixed.